packages/cloud_firestore/android/src/main/java/io/flutter/plugins/firebase/cloudfirestore/CloudFirestorePlugin.java

```diff
@@ -574,6 +576,8 @@ protected void writeValue(ByteArrayOutputStream stream, Object value) {
     } else if (value instanceof DocumentReference) {
       stream.write(DOCUMENT_REFERENCE);
       writeBytes(stream, ((DocumentReference) value).getPath().getBytes(UTF8));
+    } else if (value instanceof Blob) {
+      super.writeValue(stream, ((Blob) value).toBytes());
```

Comment:
I'd expect a similar change to the iOS side (https://github.com/flutter/plugins/blob/master/packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m#L146)

Reply:
I will have to learn myself some objective-c first but I think that should be no problem for this task :)